### ISSUE TITLE
include / extends

### ISSUE THREAD

**Issue by PastorBones** [everzet/jade.php#12]
Will the partials and inheritance features be included in this project at some point?

**Comment by seleckis** [comment]
+1
Especially "extends"

**Comment by JonniK** [comment]
+1
need this feature

**Comment by takahiro47** [comment]
+1
need this feature

**Comment by dreamline2** [comment]
+1
need this feature and add block

**Comment by oytunistrator** [comment]
+1
extends doesnt work. i need this future

```
extends ./includes/template.jade

block title
  title Jade Test Page
```

**Comment by herrbischoff** [comment]
All the +1's are doing nothing to move this forward. You do realize that the last commit to this repository was on Nov 6, 2010? How about offering the developer to lend a hand instead. Or look at the [fork list](https://github.com/everzet/jade.php/network) to find a more recent fork.

Especially to all those who wrote you "need" certain functionality: if you really _need_ this, you should consider either sponsoring the developer or hiring another developer to add the functionality you _need_. Developers _have to_ make a living as well and that usually involves money. @everzet's email address can be found in his profile. I'm willing to bet that he'll be more than happy to add all your requests for an appropriate consideration. If you're truly dependent upon this functionality, then this is a no-brainer.

This is intended to be a little on the snarky side. Please read this for context: http://www.catb.org/esr/faqs/smart-questions.html

> It sometimes looks like we're reflexively rude to newbies and the ignorant. But this isn't really true.
> 
> What we are, unapologetically, is hostile to people who seem to be unwilling to think or to do their own homework before asking questions. People like that are time sinks — they take without giving back, and they waste time we could have spent on another question more interesting and another person more worthy of an answer. We call people like this “losers” (and for historical reasons we sometimes spell it “lusers”).
> 
> We realize that there are many people who just want to use the software we write, and who have no interest in learning technical details. For most people, a computer is merely a tool, a means to an end; they have more important things to do and lives to live. We acknowledge that, and don't expect everyone to take an interest in the technical matters that fascinate us. Nevertheless, our style of answering questions is tuned for people who do take such an interest and are willing to be active participants in problem-solving. That's not going to change. Nor should it; if it did, we would become less effective at the things we do best.
> 
> We're (largely) volunteers. We take time out of busy lives to answer questions, and at times we're overwhelmed with them. So we filter ruthlessly. In particular, we throw away questions from people who appear to be losers in order to spend our question-answering time more efficiently, on winners.
> 
> If you find this attitude obnoxious, condescending, or arrogant, check your assumptions. We're not asking you to genuflect to us — in fact, most of us would love nothing more than to deal with you as an equal and welcome you into our culture, if you put in the effort required to make that possible. But it's simply not efficient for us to try to help people who are not willing to help themselves. It's OK to be ignorant; it's not OK to play stupid.

In other words: "likes" and "upvotes" do not magically make anything appear. Chin up, don't be "lusers".
